docs/content/docs/ops/state/checkpoints.md

```diff
@@ -175,6 +175,35 @@ above).
 $ bin/flink run -s :checkpointMetaDataPath [:runArgs]
 ```
 
+### Checkpoint/Savepoint guarantees
+Not always the snapshot can be successfully restored. 
+The following list contains operations which can be done without recovery lose depends on the type of the snapshot.
+
+| Operation                                             | Canonical Savepoint | Native Savepoint         | Aligned Checkpoint       | Unaligned Checkpoint     |
+|:------------------------------------------------------|:--------------------|:-------------------------|:-------------------------|:-------------------------|
+| Statebackend change                                   | Supported           | NOT Supported            | NOT Supported            | NOT Supported            |
+| Self-contained and relocatable                        | Supported           | Supported                | NOT Supported            | NOT Supported            |
```

Comment:
From current implementation, can current RocksDB native savepoint supports relocatable? I doubted this.

Reply:
I checked with latest master branch, and the native checkpoint works fine. Sorry to bother here.

Reply:
Don't worry. Thanks, for rechecking it. I am glad that everything is correct.